app/scripts/modules/netflix/fastProperties/modal/wizard/fastPropertyForm.directive.html

```diff
@@ -44,13 +44,14 @@
     <div class="form-group row">
       <div class="col-sm-3 sm-label-right">Value</div>
       <div class="col-sm-9">
-        <input type="text"
-               name="value"
-               class="form-control input-sm "
-               data-purpose="fastProperty-value"
-               ng-model="newFP.property.value"
-               placeholder="Enter a fast property value"
-               required/>
+        <textarea
+              style="resize: vertical;"
+              type="text"
```

Comment:
I don't think `type` is needed on a `textarea`.

Nice on the `resize: vertical` - I didn't know about that.